jobs/generation/Utilities.groovy

```diff
@@ -140,27 +173,27 @@ class Utilities {
         }
 
         def passCI = ''
-        if (project == 'corefx' && branch == 'master') {
+        if (project == 'corefx' && (branch == 'master' || branch == 'release/3.0')) {
           // Master uses Arcade for build and requires --ci option to be passed
           passCI = '--ci'
         }
 
         def authorsOpts = '/p:Authors=Tizen'
 
         if (project == 'coreclr') {
-          shell("${dockerCommand} ./build.sh cross \${config} \${targetArch} cmakeargs -DFEATURE_GDBJIT=TRUE stripSymbols -PortableBuild=false -- \${buildIdOpts} \${stableOpts} \${packageOpts} ${authorsOpts}")
+          shell("${dockerCommand} ./build.sh cross \${config} \${targetArch} cmakeargs -DFEATURE_GDBJIT=TRUE cmakeargs -DFEATURE_PREJIT=true stripSymbols -PortableBuild=false -- \${buildIdOpts} \${stableOpts} \${packageOpts} ${authorsOpts}")
```

Comment:
I think we should add FEATURE_NGEN_RELOCS_OPTIMIZATIONS here too, so that nugets are consistent with rpms

Reply:
done